app/views/tag/contributors.html.erb

```diff
@@ -3,37 +3,58 @@
   <% if @tag %>
     <div id="note-graph" style="height:100px;"></div>
 
+    <script src="https://cdn.jsdelivr.net/npm/frappe-charts@0.0.8/dist/frappe-charts.min.iife.js"></script>
```

Comment:
Hi @Souravirus ! 
Do you think we should put this in `bower.json` file ? See this line : 
https://github.com/publiclab/plots2/blob/d37c7c0438fd655f0de5bed6b0163764e4c144a6/bower.json#L28

The existing **chart.js** library is used in this way . I think this will cause fast rendering of graphs and also will help us to easily manage newer versions . 
What do you think ?

Reply:
Okay makes sense 👍 👍 👍  !